backend/src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -31,24 +37,33 @@
 public class EndpointResources {
 
     private static final Logger LOGGER = Logger.getLogger(EndpointResources.class);
+    public static final String OB_PROCESSOR_ID = "processorId";
+    public static final String OB_PROCESSOR_NAME = "processorname"; // Must be all lower for the filter.
+    public static final String SLACK = "Slack";
 
     @Inject
     EntityManager entityManager;
 
+    @ConfigProperty(name = "ob.enabled", defaultValue = "false")
+    boolean obEnabled;
+
+    @Inject
+    @RestClient
+    BridgeApiService bridgeApiService;
+
+    @Inject
+    Bridge bridge;
+
+    @Inject
+    BridgeAuth bridgeAuth;
```

Comment:
`EndpointResources` is supposed to be a repository for endpoints and only deal with DB queries. Can all the new code based on `BridgeApiService` be moved one layer above?